pages/cadastro/index.public.js

```diff
@@ -1,4 +1,15 @@
-import { Box, Button, DefaultLayout, Flash, FormControl, Heading, PasswordInput, TextInput } from '@/TabNewsUI';
+import {
+  Box,
+  Button,
+  DefaultLayout,
+  Flash,
+  FormControl,
+  Heading,
+  PasswordInput,
+  TextInput,
+  Link,
+  Text,
```

Comment:
```suggestion
  Link,
  PasswordInput,
  Text,
  TextInput,
```

Já que vai precisar mudar outras coisas no código, vamos tentar deixar as importações organizadas para exigir menos mudanças quando adotarmos algum plugin que faça isso automaticamente.